api-spec/spec.yaml

```diff
@@ -127,6 +119,29 @@ paths:
           description: An error occured.
           schema:
             $ref: '#/definitions/exception'
+  /search:
+    post:
+      summary: >-
+        Retrieve items matching filters. The request is a string that matches
+        the GET query exactly. POST is primarily to be used when the request
+        length exceeds GET maximums (which is not uncommon with geospatial
+        queries)
```

Comment:
I think I took a note that the group discussed flipping this around so POST should be prioritized over GET? If so we should flip this language around.

Reply:
You are right. I missed that comment in my updates.